src/components/NetAbstract/index.jsx

```diff
@@ -156,8 +158,8 @@ const NetAbstract = () => {
                 <TableCell weighting={0.2} style={styles.tableHeader}>Times</TableCell>
               </TableHeader>
               <TableBody>
-                <DataTableCell weighting={0.8} getContent={(r) => (<Text style={styles.tableRowLabelSingleLine}>{formatUrl(r.key)}</Text>)} />
-                <DataTableCell weighting={0.2} style={styles.tableRowValue} getContent={(r) => `${r.value} ms`} />
+                <DataTableCell weighting={0.8} style={styles.tableRowLabel} getContent={(r) => formatUrl(r.key)} />
+                <DataTableCell weighting={0.2} style={styles.tableRowValue} getContent={(r) => formatNumber(r.value)} />
```

Comment:
@seazon did I remove this unit accidentally?

Reply:
@XERO75  yes, I will recover it in another PR.